src/vs/workbench/contrib/terminal/browser/terminalProcessExtHostProxy.ts

```diff
@@ -140,6 +140,10 @@ export class TerminalProcessExtHostProxy extends Disposable implements ITerminal
 		// No-op
 	}
 
+	async setNextCommandId(commandLine: string, commandId: string): Promise<void> {
+		// No-op
+	}
```

Comment:
This isn't great that there are a bunch of no-op ones. That's due to being on `ITerminalChildProcess`?

Could we put it on `IPtyService` instead?